Client/Frontend/Strings.swift

```diff
@@ -1624,6 +1624,47 @@ extension String {
                     comment: "Title for the page where the user sign in to their Firefox Sync account.")
             }
         }
+
+        public struct Search {
+            public static let Title = MZLocalizedString(
```

Comment:
Are those strings really needed for 121? String freeze for that version was yesterday.

Reply:
String freeze for that version was pushed to Monday due to some confusion with stuff. So we can still get these in. Yes. :D